release/BUILD

```diff
@@ -182,6 +186,34 @@ py_test(
     ],
 )
 
+
+####
+# AIR smoke tests
```

Comment:
Nice, I'm wondering if we should aim to have one smoke test per release test (and maybe dedup the ones that are running multiple configurations). Thoughts?

Reply:
I guess we should discuss which release tests - just all benchmarks for the start? Or just anything in air_tests?

Reply:
Yes I was thinking all of the AIR tests which are mostly benchmarks anyways.